ivy/functional/ivy/general.py

```diff
@@ -2672,6 +2727,290 @@ def inplace_increment(
     return current_backend(x).inplace_increment(x, val)
 
 
+@to_native_arrays_and_back
```

Comment:
Please revert this change as it's not related to the function you're reformatting